plugins/node/opentelemetry-instrumentation-aws-lambda/test/integrations/lambda-handler.test.ts

```diff
@@ -577,6 +579,46 @@ describe('lambda handler', () => {
       );
       assert.strictEqual(span.parentSpanId, sampledHttpSpanContext.spanId);
     });
+
+    it('takes sampled custom context over sampled lambda context if "customEventContextHandler" function is defined', async () => {
+      process.env[traceContextEnvironmentKey] = sampledAwsHeader;
+      const parentSpanId = 'a0c6e3ad47f88e86';
+      const traceId = '7e41c5061d179773d209d162a47d98d6';
+      const sampledTrace = `00-${traceId}-${parentSpanId}-01`;
+      const customHandler = (event: any): OtelContext => {
+        return propagation.extract(
+          context.active(),
+          event.traceDetails
+        );
+      }
+
+      initializeHandler('lambda-test/async.handler', {
+        disableAwsContextPropagation: true,
+        customEventContextHandler: customHandler,
+      });
+
+      const otherEvent = {
+        traceDetails: {
```

Comment:
rename `traceDetails` to `contextCarrier`

Reply:
Got it